custom_components/localtuya/translations/en.json

```diff
@@ -40,7 +40,10 @@
                     "voltage": "Voltage",
                     "open_cmd": "Open Command",
                     "close_cmd": "Close Command",
-                    "stop_cmd": "Stop Command"
+                    "stop_cmd": "Stop Command",
+                    "get_position_dps": "Get Position DPS",
```

Comment:
You don't need to include `_DPs` or `DPS` here (update in code as well). It's implicit. It would also have been just `DP` for DataPoint, whereas DPS is DataPoints.